apps/gitness/src/pages-v2/pull-request/stores/pull-request-list-store.tsx

```diff
@@ -80,7 +82,7 @@ export const usePullRequestListStore = create<PullRequestListStore>(set => ({
     set({
       pullRequests: transformedPullRequests,
       totalItems: parseInt(headers?.get(PageResponseHeader.xTotal) || '0'),
-      pageSize: parseInt(headers?.get(PageResponseHeader.xPerPage) || '30')
+      pageSize: parseInt(headers?.get(PageResponseHeader.xPerPage) || '10')
```

Comment:
not sure but should we pass the `pageSize` variable as fallback instead of 10 here as well like we did 
`  pageSize: parseInt(headers?.get(PageResponseHeader.xPerPage) ?? String(pageSize))`